prisma/migrations/20240317225208_db_schema_v1/migration.sql

```diff
@@ -0,0 +1,77 @@
+-- CreateTable
+CREATE TABLE "user" (
+    "id" SERIAL NOT NULL,
+    "username" VARCHAR(20) NOT NULL,
+    "passwordHash" TEXT NOT NULL,
+    "salt" TEXT NOT NULL,
+    "role" TEXT NOT NULL DEFAULT 'user',
+
+    CONSTRAINT "user_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "movie" (
+    "id" SERIAL NOT NULL,
+    "title" VARCHAR(40) NOT NULL,
+    "summary" VARCHAR(100) NOT NULL,
+    "videoUrl" TEXT NOT NULL,
+    "coverImageUrl" TEXT NOT NULL,
+    "numberOfStreams" INTEGER NOT NULL,
+    "genres" TEXT[],
+    "mainCasts" TEXT[],
+    "releaseYear" VARCHAR(4) NOT NULL,
+    "countryOfOrigin" VARCHAR(20) NOT NULL,
+
+    CONSTRAINT "movie_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "_userWatchList" (
```

Comment:
Starting a table name with an _ is generally not recommended. Also consistency, you have tables that don't begin with an _ and then you have tables that do. It can be misleading. Also the name feels weird and a bit confusing
```suggestion
CREATE TABLE "user_movie_list" (
```

Reply:
it a prisma standard, it is auto-generated the table is used to represent the relationship between two tables. i will not query that particular table directly,